LandingZone/src/python/landing_step.py

```diff
@@ -92,7 +92,7 @@ def main():
 		usage()
 		sys.exit(1)
 
-	setupLLZ(landingZone,localLandingZone)
+	setupLLZ(landingZone+'/batch_id='+batchID,localLandingZone)
```

Comment:
I thought the data wasn't split up into batches at this point. I suppose it depends on what we do to drop data into landing zone, which we'll need to discuss later